### PR TITLE
[Optimize] Make light schema change complete more faster

### LINKED ISSUE #6291: [Optimize] Make light schema change such as add column more faster
Many OLAP System like kudu, hive, add column operation is very fast while add column in doris waste much time. in our environment, add column is normal use case, and user want to see the result in short time, so I think we can refactor some code to make make light schema change more faster

### PR DESCRIPTION
**Pull request by caiconghui**
## Proposed changes

Describe the big picture of your changes here to communicate to the maintainers why we should accept this pull request. If it fixes a bug or resolves a feature request, be sure to link to that issue.

## Types of changes

What types of changes does your code introduce to Doris?
_Put an `x` in the boxes that apply_

- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation Update (if none of the other choices apply)
- [ ] Code refactor (Modify the code structure, format the code, etc...)
- [x] Optimization. Including functional usability improvements and performance improvements.
- [ ] Dependency. Such as changes related to third-party components.
- [ ] Other.

## Checklist

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code._

- [x] I have created an issue on (Fix #6291 ) and described the bug/feature there in detail
- [ ] Compiling and unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] If these changes need document changes, I have updated the document
- [ ] Any dependent changes have been merged

## Further comments

If this is a relatively large or complex change, kick off the discussion at dev@doris.apache.org by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

File: be/src/common/config.h [hunk 1]
```diff
@@ -87,7 +87,7 @@ CONF_Int32(clear_transaction_task_worker_count, "1");
 // the count of thread to delete
 CONF_Int32(delete_worker_count, "3");
 // the count of thread to alter table
-CONF_Int32(alter_tablet_worker_count, "3");
+CONF_Int32(alter_tablet_worker_count, "6");
```
Comment (morningman) [inline be/src/common/config.h:90]: better not changing the default value if it is not that much necessary.

**Review by morningman — COMMENTED** [review]
I don't get it, why it get faster than before?
I notice that you put a limit number of active jobs in a new thread pool and let them run concurrently?

File: be/src/common/config.h (hunk 1 above)
Comment (caiconghui) [reply be/src/common/config.h:90]: ok. maybe it is better decided by user's business scenario, I will restore the value to its original state

**Comment by caiconghui** [comment]
> I don't get it, why it get faster than before?
> I notice that you put a limit number of active jobs in a new thread pool and let them run concurrently?

on our production environment, there are usually some tables added columns at the same time, and now these table are loaded data all time by stream load, so We hope that adding column operation can be complished as soon as possible, or stream load can be more slower when it should write two copies of data. now, it waste time to sleep and schema change jobs will affect each other, so here I change the interval for schema change check and add threadpool to make schema change jobs wouldn't affect each other.

**Review by morningman — APPROVED** [review]
LGTM

**Pull request merged**